classes/module/Module.php

```diff
@@ -219,6 +222,11 @@ abstract class ModuleCore implements ModuleInterface
     const CACHE_FILE_TRUSTED_MODULES_LIST = '/config/xml/trusted_modules_list.xml';
     const CACHE_FILE_UNTRUSTED_MODULES_LIST = '/config/xml/untrusted_modules_list.xml';
 
+    public const MULTISTORE_COMPATIBILITY_NO = 0;
+    public const MULTISTORE_COMPATIBILITY_NOT_CONCERNED = 1;
+    public const MULTISTORE_COMPATIBILITY_PARTIAL = 2;
+    public const MULTISTORE_COMPATIBILITY_YES = 3;
```

Comment:
This will allow to add intermediate states in the future
```suggestion
    public const MULTISTORE_COMPATIBILITY_NO = -20;
    public const MULTISTORE_COMPATIBILITY_NOT_CONCERNED = -10;
    public const MULTISTORE_COMPATIBILITY_UNKNOWN = 0;
    public const MULTISTORE_COMPATIBILITY_PARTIAL = 10;
    public const MULTISTORE_COMPATIBILITY_YES = 20;
```